contracts/src/v0.8/functions/tests/v1_X/FunctionsSubscriptions.t.sol

```diff
@@ -352,10 +366,10 @@ contract FunctionsSubscriptions_OnTokenTransfer is FunctionsSubscriptionSetup {
     uint96 subscriptionBalanceBefore = s_functionsRouter.getSubscription(s_subscriptionId).balance;
```

Comment:
Is this always zero now?

Reply:
Yeah, lemme remove it so it's not extra noise